packages/config/config/devices/0x010f/templates/fgwpx-1x1.json

```
@@ -0,0 +1,489 @@
+// Template for FGWPX-111 and FGWPX-121
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"description": "Metered Wall Plug Switch",
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "Power Failure",
+			"description": "How the Wall Plug will react in the event of power supply failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Switched OFF",
+					"value": 0
+				},
+				{
+					"label": "Restore State",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"label": "Overload",
+			"description": "Turn off the controlled device in case of exceeding the defined power",
+			"valueSize": 2,
+			"unit": "0.1 Watts",
+			"minValue": 0,
+			"maxValue": 18000,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Overload Disabled",
+					"value": 0
+				}
+			]
+		},
+		"11": {
+			"label": "Power Reporting",
+			"description": "Minimum change in active power consumption to trigger",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disables power reporting",
+					"value": 0
+				}
+			]
+		},
+		"12": {
+			"label": "Energy Reporting Threshold",
+			"valueSize": 2,
+			"unit": "0.01 kWh",
+			"minValue": 0,
+			"maxValue": 500,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disables energy reporting",
+					"value": 0
+				}
+			]
+		},
+		"13": {
+			"label": "Periodic Power Report",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 0,
+			"maxValue": 32400,
+			"defaultValue": 3600,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disables reporting",
```

```suggestion
					"label": "Disable reporting",
```
Options should be formulated in an active way, not describe what they do.